app/lib/methods/handleAudioMedia.ts

```diff
@@ -0,0 +1,102 @@
+import { AVPlaybackStatus, Audio, InterruptionModeAndroid, InterruptionModeIOS } from 'expo-av';
+import { Sound } from 'expo-av/build/Audio/Sound';
+
+const mode = {
+	allowsRecordingIOS: false,
+	playsInSilentModeIOS: true,
+	staysActiveInBackground: true,
+	shouldDuckAndroid: true,
+	playThroughEarpieceAndroid: false,
+	interruptionModeIOS: InterruptionModeIOS.DoNotMix,
+	interruptionModeAndroid: InterruptionModeAndroid.DoNotMix
+};
+
+class HandleAudioMedia {
```

Comment:
Why a class?

Reply:
I decided to create a class to unify the data and behaviors inside it because probably we wouldn't reuse the same methods in another type of file. Also, IMHO this is easier to understand 😁

Reply:
To do without a singleton will look like the handleMediaDownload (that I did too).